### ISSUE TITLE
Draft Co-Deign Group expectations email to SMEs

### ISSUE THREAD

**Issue by rishalee** [HHS/simpler-grants-gov#1642]
### Summary

SMEs have indicated willingness to recruit within their network for the Co-Design group, but are unwilling to reach out to their community until receiving more information about what the co design group will entail (including such items as time commitment, compensation, etc)

- Write email to SMEs detailing group expectations and compensation so they can reach out and provide info to contacts who would be good fits for the Co-Design Group. Message will incorporate info from #1441 & #1665 
- Receive approval from MH to pass to HHS Design/Comms Leads
- Incorporate edits from HHS Design/Comms Leads (if necessary)
- Receive approval from HHS to send to HHS Project leads
- Incorporate edits from HHS Project leads (if necessary)

### Acceptance criteria

- [ ] Email with description has been drafted
- [ ] MH approval 
- [ ] HHS Design/Comms leads approval
- [ ] Final approval from HHS Project Leads

**Comment by andycochran** [comment]
- Noting that we've been in contact with the design team at USDR who have also assembled a "Co-Design Group" of sorts. They meet regularly with a group of users for product updates, feedback, and research. 
- We've discussed how both of our projects are interested in sharing findings, staying aware of each other's work/roadmaps as to avoid duplication of efforts, and looking for opportunities to collaborate more directly. 
- USRD will be interested to hear about how our Co-Design Group shapes up, and may have insight on the relationships we build, particularly with grantors they are already speaking with. 
- We've established a quarterly check-in call, the next of which is April 26th when we will be sharing our roadmaps (and intro-ing developers who may be interested in our FOSS onboarding)

cc @margaretspring @lucasmbrown-usds
